File: FrameIncam.Domains/Common/CustomerProfile.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FrameIncam.Domains.Common
{
    public class CustomerProfile
    {
        public int? customerId { get; set; }
        public string customerAddressLine1 { get; set; }
        public string customerAddressLine2 { get; set; }
        public string customerAltMobileNo { get; set; }
        public string customerCity { get; set; }
        public string customerCode { get; set; }
        public string customerEmail { get; set; }
        public string customerMobileNo { get; set; }
        public string customerName { get; set; }
        public string customerPin { get; set; }
    }
}

File: FrameIncam.Client.FreeLancer/ng/domains/common/customerProfile.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var common = p_models.namespace("fincam.common");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: common,
                name: "customerProfile",
                base: {
                    namespace: fincam,
                    name: "model"
                },
                properties: {
                    customerId: "int",
                    customerAddressLine1: "string",
                    customerAddressLine2: "string",
                    customerAltMobileNo: "string",
                    customerCity: "string",
                    customerCode: "string",
                    customerEmail: "string",
                    customerMobileNo: "string",
                    customerName: "string"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));